app/api/chat/route.ts

```diff
@@ -139,8 +137,8 @@ export async function POST(req: NextRequest) {
     // "Pick" the answer from the retrieval chain output object and stream it as bytes.
     const outputChain = RunnableSequence.from([
       conversationalRetrievalChain,
-      new RunnablePick({ keys: "answer" }),
-      new HttpResponseOutputParser({ contentType: "text/plain" }),
+      new RunnablePick({ keys: 'answer' }),
```

Comment:
If you wouldn't mind - `conversationalRetrievalChain.pick('answer')` would be a little nicer here